Make code golint compliant

```bash
$ make golint
Running lint
env GO111MODULE=off /usr/bin/go get golang.org/x/lint/golint
/home/sumacheb/go/bin/golint -set_exit_status ./...
server/plugin.go:26:2: exported const PostMeetingKey should have comment (or a comment on this block) or be unexported
server/plugin.go:35:6: exported type IClient should have comment or be unexported
server/plugin.go:44:6: exported type Plugin should have comment or be unexported
server/plugin.go:57:1: exported method Plugin.OnActivate should have comment or be unexported
Found 4 lint suggestions; failing.
make: *** [Makefile:80: golint] Error 1
```

--- a/server/plugin.go
+++ b/server/plugin.go
@@ -23,13 +23,13 @@ import (
 )
 
 const (
-	POST_MEETING_KEY               = "post_meeting_"
-	POST_MEETING_TYPE              = "custom_s4b"
-	POST_MEETING_OVERRIDE_USERNAME = "Skype for Business Plugin"
-	NEW_APPLICATION_USER_AGENT     = "mm_skype4b_plugin"
-	NEW_APPLICATION_CULTURE        = "en-US"
-	WS_EVENT_AUTHENTICATED         = "authenticated"
-	ROOT_URL_KEY                   = "root_url"
+	PostMeetingKey              = "post_meeting_"
+	PostMeetingType             = "custom_s4b"
+	PostMeetingOverrideUsername = "Skype for Business Plugin"
+	NewApplicationUserAgent     = "mm_skype4b_plugin"
+	NewApplicationCulture       = "en-US"
+	WsEventAuthenticated        = "authenticated"
+	RootURLKey                  = "root_url"
 )
 
 type IClient interface {
@@ -77,7 +77,7 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 	case "/api/v1/create_meeting_in_server_version":
 		p.handleCreateMeetingInServerVersion(w, r)
 	case "/api/v1/client_id":
-		p.handleClientId(w, r)
+		p.handleClientID(w, r)
 	case "/api/v1/auth":
 		p.handleAuthorizeInADD(w, r)
 	case "/api/v1/auth_redirect":
@@ -93,45 +93,45 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 
 func (p *Plugin) handleAuthorizeInADD(w http.ResponseWriter, r *http.Request) {
 
-	userId := r.URL.Query().Get("mattermost_user_id")
+	userID := r.URL.Query().Get("mattermost_user_id")
 
-	if userId == "" {
+	if userID == "" {
 		fmt.Println("Not authorized")
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
 
-	encodedAuthUrl := r.URL.Query().Get("navigateTo")
-	if encodedAuthUrl == "" {
+	encodedAuthURL := r.URL.Query().Get("navigateTo")
+	if encodedAuthURL == "" {
 		fmt.Println("Url Param 'navigateTo' is missing")
 		http.Error(w, "Url Param 'navigateTo' is missing", http.StatusBadRequest)
 		return
 	}
 
-	authUrl, err := url.QueryUnescape(encodedAuthUrl)
+	authURL, err := url.QueryUnescape(encodedAuthURL)
 	if err != nil {
 		fmt.Println(err.Error())
 		http.Error(w, "cannot decode url", http.StatusBadRequest)
 		return
 	}
 
-	authUrlValues, err := url.ParseQuery(authUrl)
+	authURLValues, err := url.ParseQuery(authURL)
 	if err != nil {
 		fmt.Println("cannot parse url")
 		http.Error(w, "cannot parse url", http.StatusBadRequest)
 		return
 	}
 
-	state := authUrlValues.Get("state")
+	state := authURLValues.Get("state")
 	if state == "" {
 		fmt.Println("Url Param 'state' is missing")
 		http.Error(w, "Url Param 'state' is missing", http.StatusBadRequest)
 		return
 	}
 
-	p.API.KVSet(state, []byte(strings.TrimSpace(userId)))
+	p.API.KVSet(state, []byte(strings.TrimSpace(userID)))
 
-	http.Redirect(w, r, authUrl, http.StatusFound)
+	http.Redirect(w, r, authURL, http.StatusFound)
 }
 
 func (p *Plugin) completeAuthorizeInADD(w http.ResponseWriter, r *http.Request) {
@@ -150,13 +150,15 @@ func (p *Plugin) completeAuthorizeInADD(w http.ResponseWriter, r *http.Request) 
 		return
 	}
 
-	userId, err := p.API.KVGet(state)
+	userID, err := p.API.KVGet(state)
 
 	if err != nil {
 		fmt.Println(err.Message)
 		http.Error(w, "cannot get stored state", http.StatusBadRequest)
 		return
-	} else if userId == nil {
+	}
+
+	if userID == nil {
 		http.Error(w, "missing stored state", http.StatusBadRequest)
 		return
 	}
@@ -166,11 +168,11 @@ func (p *Plugin) completeAuthorizeInADD(w http.ResponseWriter, r *http.Request) 
 		fmt.Println("cannot delete stored state", err)
 	}
 
-	p.API.PublishWebSocketEvent(WS_EVENT_AUTHENTICATED, map[string]interface{}{
+	p.API.PublishWebSocketEvent(WsEventAuthenticated, map[string]interface{}{
 		"token": idToken,
 		"state": state,
 	}, &model.WebsocketBroadcast{
-		UserId: strings.TrimSpace(string(userId)),
+		UserId: strings.TrimSpace(string(userID)),
 	})
 
 	html := `
@@ -195,11 +197,11 @@ func (p *Plugin) completeAuthorizeInADD(w http.ResponseWriter, r *http.Request) 
 	w.Write([]byte(html))
 }
 
-func (p *Plugin) handleClientId(w http.ResponseWriter, r *http.Request) {
+func (p *Plugin) handleClientID(w http.ResponseWriter, r *http.Request) {
 
-	userId := r.Header.Get("Mattermost-User-Id")
+	userID := r.Header.Get("Mattermost-User-Id")
 
-	if userId == "" {
+	if userID == "" {
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
@@ -218,8 +220,8 @@ func (p *Plugin) handleClientId(w http.ResponseWriter, r *http.Request) {
 
 func (p *Plugin) handleProductType(w http.ResponseWriter, r *http.Request) {
 
-	userId := r.Header.Get("Mattermost-User-Id")
-	if userId == "" {
+	userID := r.Header.Get("Mattermost-User-Id")
+	if userID == "" {
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
@@ -244,19 +246,21 @@ func (p *Plugin) handleProductType(w http.ResponseWriter, r *http.Request) {
 
 func (p *Plugin) handleRegisterMeetingFromOnlineVersion(w http.ResponseWriter, r *http.Request) {
 
-	userId := r.Header.Get("Mattermost-User-Id")
-	if userId == "" {
+	userID := r.Header.Get("Mattermost-User-Id")
+	if userID == "" {
 		fmt.Println("Request doesn't have Mattermost-User-Id header")
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
 
-	user, appErr := p.API.GetUser(userId)
+	user, appErr := p.API.GetUser(userID)
 	if appErr != nil {
 		fmt.Println(appErr.Error())
 		http.Error(w, appErr.Error(), appErr.StatusCode)
 		return
-	} else if user == nil {
+	}
+
+	if user == nil {
 		fmt.Println("User is nil")
 		http.Error(w, "User is nil", http.StatusUnauthorized)
 		return
@@ -267,7 +271,9 @@ func (p *Plugin) handleRegisterMeetingFromOnlineVersion(w http.ResponseWriter, r
 		fmt.Println("Cannot fetch configuration")
 		http.Error(w, "Forbidden", http.StatusForbidden)
 		return
-	} else if config.ProductType == productTypeServer {
+	}
+
+	if config.ProductType == productTypeServer {
 		fmt.Println("Server version is set")
 		http.Error(w, "Forbidden", http.StatusForbidden)
 		return
@@ -293,13 +299,13 @@ func (p *Plugin) handleRegisterMeetingFromOnlineVersion(w http.ResponseWriter, r
 		UserId:    user.Id,
 		ChannelId: req.ChannelId,
 		Message:   fmt.Sprintf("Meeting started at %s.", req.MeetingURL),
-		Type:      POST_MEETING_TYPE,
+		Type:      PostMeetingType,
 		Props: map[string]interface{}{
 			"meeting_id":        req.MeetingId,
 			"meeting_link":      req.MeetingURL,
 			"meeting_personal":  req.Personal,
 			"meeting_topic":     req.Topic,
-			"override_username": POST_MEETING_OVERRIDE_USERNAME,
+			"override_username": PostMeetingOverrideUsername,
 			"meeting_status":    "STARTED",
 			"from_webhook":      "true",
 			"override_icon_url": path.Join(*serverConfiguration.ServiceSettings.SiteURL, "plugins", manifest.ID, "api", "v1", "assets", "profile.png"),
@@ -313,7 +319,7 @@ func (p *Plugin) handleRegisterMeetingFromOnlineVersion(w http.ResponseWriter, r
 		return
 	}
 
-	if appErr = p.API.KVSet(fmt.Sprintf("%v%v", POST_MEETING_KEY, req.MeetingId), []byte(post.Id)); appErr != nil {
+	if appErr = p.API.KVSet(fmt.Sprintf("%v%v", PostMeetingKey, req.MeetingId), []byte(post.Id)); appErr != nil {
 		fmt.Println(appErr.Error())
 		http.Error(w, appErr.Error(), appErr.StatusCode)
 		return
@@ -330,8 +336,8 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 		return
 	}
 
-	userId := r.Header.Get("Mattermost-User-Id")
-	if userId == "" {
+	userID := r.Header.Get("Mattermost-User-Id")
+	if userID == "" {
 		mlog.Error("Request doesn't have Mattermost-User-Id header")
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
@@ -339,13 +345,15 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 
 	var user *model.User
 	var appError *model.AppError
-	user, appError = p.API.GetUser(userId)
+	user, appError = p.API.GetUser(userID)
 	if appError != nil {
 		mlog.Error("Error getting user: " + appError.Error())
 		http.Error(w, appError.Error(), appError.StatusCode)
 		return
-	} else if user == nil {
-		mlog.Error("User with that id doesn't exist: " + userId)
+	}
+
+	if user == nil {
+		mlog.Error("User with that id doesn't exist: " + userID)
 		http.Error(w, "User is nil", http.StatusUnauthorized)
 		return
 	}
@@ -355,13 +363,15 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 		mlog.Error("Error decoding JSON body: " + err.Error())
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
-	} else if _, err := p.API.GetChannelMember(req.ChannelId, user.Id); err != nil {
+	}
+
+	if _, err := p.API.GetChannelMember(req.ChannelId, user.Id); err != nil {
 		mlog.Error("Error getting channel member: " + err.Error())
 		http.Error(w, "Forbidden", http.StatusForbidden)
 		return
 	}
 
-	applicationState, apiErr := p.fetchOnlineMeetingsUrl()
+	applicationState, apiErr := p.fetchOnlineMeetingsURL()
 	if apiErr != nil {
 		mlog.Error("Error fetching meetings resource url: " + apiErr.Message)
 		http.Error(w, apiErr.Message, http.StatusInternalServerError)
@@ -388,12 +398,12 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 		UserId:    user.Id,
 		ChannelId: req.ChannelId,
 		Message:   fmt.Sprintf("Meeting started at %s.", newMeetingResponse.JoinUrl),
-		Type:      POST_MEETING_TYPE,
+		Type:      PostMeetingType,
 		Props: map[string]interface{}{
 			"meeting_id":        newMeetingResponse.MeetingId,
 			"meeting_link":      newMeetingResponse.JoinUrl,
 			"meeting_personal":  req.Personal,
-			"override_username": POST_MEETING_OVERRIDE_USERNAME,
+			"override_username": PostMeetingOverrideUsername,
 			"meeting_topic":     "Meeting created by " + user.Username,
 			"meeting_status":    "STARTED",
 			"from_webhook":      "true",
@@ -408,7 +418,7 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 		return
 	}
 
-	appErr = p.API.KVSet(fmt.Sprintf("%v%v", POST_MEETING_KEY, newMeetingResponse.MeetingId), []byte(post.Id))
+	appErr = p.API.KVSet(fmt.Sprintf("%v%v", PostMeetingKey, newMeetingResponse.MeetingId), []byte(post.Id))
 	if appErr != nil {
 		mlog.Error("Error writing meeting id to the database: " + appErr.Error())
 		http.Error(w, appErr.Error(), http.StatusInternalServerError)
@@ -441,13 +451,13 @@ func (p *Plugin) handleProfileImage(w http.ResponseWriter, r *http.Request) {
 	io.Copy(w, img)
 }
 
-func (p *Plugin) fetchOnlineMeetingsUrl() (*ApplicationState, *APIError) {
-	rootUrl, apiErr := p.getRootUrl()
+func (p *Plugin) fetchOnlineMeetingsURL() (*ApplicationState, *APIError) {
+	rootURL, apiErr := p.getRootURL()
 	if apiErr != nil {
 		return nil, apiErr
 	}
 
-	applicationState, apiError := p.getApplicationState(*rootUrl)
+	applicationState, apiError := p.getApplicationState(*rootURL)
 	if apiError != nil {
 		return nil, apiError
 	}
@@ -455,8 +465,8 @@ func (p *Plugin) fetchOnlineMeetingsUrl() (*ApplicationState, *APIError) {
 	newApplicationResponse, err := p.client.createNewApplication(
 		applicationState.ApplicationsUrl,
 		NewApplicationRequest{
-			UserAgent:  NEW_APPLICATION_USER_AGENT,
-			Culture:    NEW_APPLICATION_CULTURE,
+			UserAgent:  NewApplicationUserAgent,
+			Culture:    NewApplicationCulture,
 			EndpointId: "123",
 		},
 		applicationState.Token,
@@ -470,10 +480,10 @@ func (p *Plugin) fetchOnlineMeetingsUrl() (*ApplicationState, *APIError) {
 	return applicationState, nil
 }
 
-func (p *Plugin) getApplicationState(discoveryUrl string) (*ApplicationState, *APIError) {
+func (p *Plugin) getApplicationState(discoveryURL string) (*ApplicationState, *APIError) {
 	config := p.getConfiguration()
 
-	DiscoveryResponse, err := p.client.performDiscovery(discoveryUrl)
+	DiscoveryResponse, err := p.client.performDiscovery(discoveryURL)
 	if err != nil {
 		return nil, &APIError{Message: "Error performing autodiscovery: " + err.Error()}
 	}
@@ -483,84 +493,86 @@ func (p *Plugin) getApplicationState(discoveryUrl string) (*ApplicationState, *A
 		return nil, &APIError{Message: "Error performing request to get authentication header: " + err.Error()}
 	}
 
-	tokenUrl, apiErr := p.extractTokenUrl(*authHeader)
+	tokenURL, apiErr := p.extractTokenURL(*authHeader)
 	if apiErr != nil {
 		return nil, apiErr
 	}
 
-	userResourceUrl := DiscoveryResponse.Links.User.Href
-	resourceName := p.extractResourceNameFromUserUrl(userResourceUrl)
-	authResponse, err := p.authenticate(*tokenUrl, resourceName, *config)
+	userResourceURL := DiscoveryResponse.Links.User.Href
+	resourceName := p.extractResourceNameFromUserURL(userResourceURL)
+	authResponse, err := p.authenticate(*tokenURL, resourceName, *config)
 	if err != nil {
 		return nil, &APIError{Message: "Error during authentication: " + err.Error()}
 	}
 
-	userResourceResponse, err := p.client.readUserResource(userResourceUrl, authResponse.Access_token)
+	userResourceResponse, err := p.client.readUserResource(userResourceURL, authResponse.Access_token)
 	if err != nil {
 		return nil, &APIError{Message: "Error reading user resource: " + err.Error()}
 	}
 
-	applicationsUrl := userResourceResponse.Links.Applications.Href
-	if applicationsUrl != "" {
+	applicationsURL := userResourceResponse.Links.Applications.Href
+	if applicationsURL != "" {
 
-		applicationsResourceName := p.extractResourceNameFromApplicationsUrl(applicationsUrl)
+		applicationsResourceName := p.extractResourceNameFromApplicationsURL(applicationsURL)
 		if applicationsResourceName != resourceName {
 			mlog.Warn("Resource from applications url is not the same as resource name from user url")
 
-			authHeader, err := p.client.performRequestAndGetAuthHeader(applicationsUrl)
+			authHeader, err := p.client.performRequestAndGetAuthHeader(applicationsURL)
 			if err != nil {
 				return nil, &APIError{
 					Message: "Error performing request to get authentication header from new resource: " + err.Error(),
 				}
 			}
 
-			tokenUrl, apiErr := p.extractTokenUrl(*authHeader)
+			tokenURL, apiErr := p.extractTokenURL(*authHeader)
 			if apiErr != nil {
 				return nil, apiErr
 			}
 
-			authResponse, err = p.authenticate(*tokenUrl, applicationsResourceName, *config)
+			authResponse, err = p.authenticate(*tokenURL, applicationsResourceName, *config)
 			if err != nil {
 				return nil, &APIError{Message: "Error during authentication in new resource: " + err.Error()}
 			}
 		}
 
 		return &ApplicationState{
-			ApplicationsUrl: applicationsUrl,
+			ApplicationsUrl: applicationsURL,
 			Resource:        applicationsResourceName,
 			Token:           authResponse.Access_token,
 		}, nil
-	} else if userResourceResponse.Links.Redirect.Href != "" {
+	}
+
+	if userResourceResponse.Links.Redirect.Href != "" {
 		return p.getApplicationState(userResourceResponse.Links.Redirect.Href)
-	} else {
-		return nil, &APIError{
-			Message: "Neither applications resource or redirect resource fetched from user resource",
-		}
+	}
+
+	return nil, &APIError{
+		Message: "Neither applications resource or redirect resource fetched from user resource",
 	}
 }
 
-func (p *Plugin) getRootUrl() (*string, *APIError) {
-	rootUrlBytes, appErr := p.API.KVGet(ROOT_URL_KEY)
+func (p *Plugin) getRootURL() (*string, *APIError) {
+	rootURLBytes, appErr := p.API.KVGet(RootURLKey)
 	if appErr != nil {
 		return nil, &APIError{Message: "Cannot fetch the root url from the database: " + appErr.Error()}
 	}
 
-	if rootUrlBytes != nil {
-		rootUrl := string(rootUrlBytes)
-		return &rootUrl, nil
+	if rootURLBytes != nil {
+		rootURL := string(rootURLBytes)
+		return &rootURL, nil
 	}
 
-	rootUrl, err := p.determineRootUrl(p.getConfiguration().Domain)
+	rootURL, err := p.determineRootURL(p.getConfiguration().Domain)
 	if err != nil {
 		return nil, err
 	}
 
-	_ = p.API.KVSet(ROOT_URL_KEY, []byte(*rootUrl))
+	_ = p.API.KVSet(RootURLKey, []byte(*rootURL))
 
-	return rootUrl, nil
+	return rootURL, nil
 }
 
-func (p *Plugin) determineRootUrl(domain string) (*string, *APIError) {
+func (p *Plugin) determineRootURL(domain string) (*string, *APIError) {
 	for _, o := range []struct {
 		url  string
 		name string
@@ -585,9 +597,9 @@ func (p *Plugin) determineRootUrl(domain string) (*string, *APIError) {
 		_, err := p.client.performDiscovery(o.url)
 		if err == nil {
 			return &o.url, nil
-		} else {
-			mlog.Warn("Error performing autodiscovery with " + o.name + " root URL: " + err.Error())
 		}
+
+		mlog.Warn("Error performing autodiscovery with " + o.name + " root URL: " + err.Error())
 	}
 
 	return nil, &APIError{
@@ -595,14 +607,14 @@ func (p *Plugin) determineRootUrl(domain string) (*string, *APIError) {
 	}
 }
 
-func (p *Plugin) extractTokenUrl(authHeader string) (*string, *APIError) {
-	webTicketUrlRegexMatch := regexp.MustCompile(`href=(.*?),`).FindStringSubmatch(authHeader)
-	if len(webTicketUrlRegexMatch) < 1 {
+func (p *Plugin) extractTokenURL(authHeader string) (*string, *APIError) {
+	webTicketURLRegexMatch := regexp.MustCompile(`href=(.*?),`).FindStringSubmatch(authHeader)
+	if len(webTicketURLRegexMatch) < 1 {
 		return nil, &APIError{
 			Message: "Cannot extract webTicket URL from WWW-AUTHENTICATE header! Full header value: " + authHeader,
 		}
 	}
-	webTicketUrl := strings.ReplaceAll(webTicketUrlRegexMatch[1], "\"", "")
+	webTicketURL := strings.ReplaceAll(webTicketURLRegexMatch[1], "\"", "")
 
 	grantTypeRegexMatch := regexp.MustCompile(`grant_type="(.*?)"`).FindStringSubmatch(authHeader)
 	if len(grantTypeRegexMatch) < 1 {
@@ -618,11 +630,11 @@ func (p *Plugin) extractTokenUrl(authHeader string) (*string, *APIError) {
 		}
 	}
 
-	return &webTicketUrl, nil
+	return &webTicketURL, nil
 }
 
-func (p *Plugin) authenticate(tokenUrl string, resourceName string, config configuration) (*AuthResponse, error) {
-	return p.client.authenticate(tokenUrl, url.Values{
+func (p *Plugin) authenticate(tokenURL string, resourceName string, config configuration) (*AuthResponse, error) {
+	return p.client.authenticate(tokenURL, url.Values{
 		"grant_type": {"password"},
 		"username":   {config.Username},
 		"password":   {config.Password},
@@ -630,14 +642,14 @@ func (p *Plugin) authenticate(tokenUrl string, resourceName string, config confi
 	})
 }
 
-func (p *Plugin) extractResourceNameFromUserUrl(userUrl string) string {
+func (p *Plugin) extractResourceNameFromUserURL(userURL string) string {
 	resourceRegex := regexp.MustCompile(`https:\/\/(.*)\/Autodiscover\/`)
-	resourceRegexMatch := resourceRegex.FindStringSubmatch(userUrl)
+	resourceRegexMatch := resourceRegex.FindStringSubmatch(userURL)
 	return resourceRegexMatch[1]
 }
 
-func (p *Plugin) extractResourceNameFromApplicationsUrl(applicationsUrl string) string {
+func (p *Plugin) extractResourceNameFromApplicationsURL(applicationsURL string) string {
 	resourceRegex := regexp.MustCompile(`https:\/\/(.*)\/ucwa\/`)
-	resourceRegexMatch := resourceRegex.FindStringSubmatch(applicationsUrl)
+	resourceRegexMatch := resourceRegex.FindStringSubmatch(applicationsURL)
 	return resourceRegexMatch[1]
 }

--- a/server/plugin_test.go
+++ b/server/plugin_test.go
@@ -128,7 +128,7 @@ func TestPlugin(t *testing.T) {
 
 			api.On("GetChannelMember", "thechannelid", "theuserid").Return(&model.ChannelMember{}, (*model.AppError)(nil))
 			api.On("CreatePost", mock.AnythingOfType("*model.Post")).Return(&model.Post{}, (*model.AppError)(nil))
-			api.On("KVSet", fmt.Sprintf("%v%v", POST_MEETING_KEY, "L30IC51J"), mock.AnythingOfType("[]uint8")).Return((*model.AppError)(nil))
+			api.On("KVSet", fmt.Sprintf("%v%v", PostMeetingKey, "L30IC51J"), mock.AnythingOfType("[]uint8")).Return((*model.AppError)(nil))
 			api.On("KVSet", "123", []byte("theuserid")).Return((*model.AppError)(nil))
 			api.On("KVGet", "123").Return([]byte("theuserid"), (*model.AppError)(nil))
 			api.On("KVDelete", "123").Return((*model.AppError)(nil))
@@ -234,7 +234,7 @@ func makeMocks(mmChannelId string, mmUser model.User, splitDomain bool) Mocks {
 	api.On("GetUser", mmUser.Id).Return(&mmUser, (*model.AppError)(nil)).Times(1)
 	api.On("GetChannelMember", mmChannelId, mmUser.Id).
 		Return(&model.ChannelMember{}, (*model.AppError)(nil)).Times(1)
-	api.On("KVGet", ROOT_URL_KEY).
+	api.On("KVGet", RootURLKey).
 		Return([]byte(discoveryUrlToReturn), (*model.AppError)(nil)).Times(1)
 	api.On("CreatePost", expectedPost).
 		Return(&model.Post{}, (*model.AppError)(nil)).Times(1)
@@ -371,8 +371,8 @@ func makeApplicationsUrl(domain string) string {
 
 func makeNewApplicationRequest() NewApplicationRequest {
 	return NewApplicationRequest{
-		UserAgent:  NEW_APPLICATION_USER_AGENT,
-		Culture:    NEW_APPLICATION_CULTURE,
+		UserAgent:  NewApplicationUserAgent,
+		Culture:    NewApplicationCulture,
 		EndpointId: "123",
 	}
 }

